packages/core/src/client/decorators/bundlerClient.ts

```diff
@@ -71,8 +83,10 @@ export type BundlerActions = {
    * @param entryPoint - the entry point address the op will be sent to
    * @returns the hash of the sent UserOperation
    */
-  sendRawUserOperation(
-    request: UserOperationRequest,
+  sendRawUserOperation<
+    TEntryPointVersion extends EntryPointVersion = DefaultEntryPointVersion
```

Comment:
this one I think we talked about async, but this method should probs not default
```suggestion
    TEntryPointVersion extends EntryPointVersion = EntryPointVersion
```